.circleci/config.yml

```diff
@@ -25,6 +25,10 @@ jobs:
       - run:
           name: Provide DevNet mnemonics
           command: echo $DEV_NET_MNEMONICS > android-sdk/src/androidTest/res/raw/dev_net_mnemonics
+          
+      - run:
```

Comment:
was there an explicit ask for us to test with root entropies ? I can add it to iOS if there was

Reply:
No, but the test keys for the android accounts on alpha use root entropies. A prior change I made preemptively updated both alpha and mobile dev to the new mnemonics when only dev net was updated. This is just to fix the alpha accounts until it gets updated to 1.2.